app/js/components/manageVisit.jsx

```diff
@@ -0,0 +1,274 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from 'react';
+import toastr from 'toastr';
+import moment from 'moment';
+import apiCall from '../utilities/apiHelper';
+import DateTimeField from 'react-bootstrap-datetimepicker';
+import { Button, Form, FormGroup, Label, Input, FormText, Table, ListGroup, ListGroupItem } from 'reactstrap';
+import { withRouter } from 'react-router'
+
+const getSuggestionValue = suggestion => suggestion.name;
+
+class ManageVisit extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      uuid: this.props.params.patentId,
+      visituuid: this.props.params.visitId,
+      display: '',
+      location: [],
+      locationName: '',
+      patientName: '',
+      format: "YYYY-MM-DD HH:mm:ss",
```

Comment:
We shall have to let the end user configure the date formats instead of hard coding them.